cms/static/sass/views/_container.scss

```diff
@@ -331,3 +331,112 @@
     }
   }
 }
+
+
+.move-xblock-modal {
+
+  button {
+    background: transparent;
+    border-color: transparent;
+    padding: 0;
+    border: none;
+  }
+
+  .breadcrumb-container {
+    margin-bottom: ($baseline/4);
+    border: 1px solid #D9D9D9;
+    padding: ($baseline/2);
+    background: #F2F8FB;
+
+    .breadcrumbs {
+      .bc-container {
+        @include font-size(14);
```

Comment:
I'm not familiar with the Studio sass, but are there variables for font sizes? Hard coding the size makes it difficult for theming.